x-pack/plugins/logstash/public/components/pipeline_editor/pipeline_editor.js

```diff
@@ -189,9 +190,15 @@ class PipelineEditorUi extends React.Component {
   };
 
   notifyOnError = err => {
-    const { notifier, licenseService } = this.props;
+    const { licenseService, toastNotifications } = this.props;
 
-    return licenseService.checkValidity().then(() => notifier.error(err));
+    return licenseService.checkValidity().then(() => {
+      toastNotifications.addError(err, {
+        title: i18n.translate('xpack.logstash.pipelineEditor.errorHandlerToastTitle', {
+          defaultMessage: 'Error'
```

Comment:
Is there anything more specific that can be added here other than the generic "Error" text?

Reply:
I tried to find some and failed. The problem is, that it seems that failure code might be called from very different type of requests, which I couldn't come up with a better wording. Maybe @ycombinator has an idea what might make more sense as a toast title here?